.github/workflows/ci.yml

```diff
@@ -13,7 +13,11 @@ jobs:
         with:
           node-version: 22
       - run: npm ci
+        env:
+          RPC_AUTH: ${{secrets.GLIF_API_TOKEN}}
```

Comment:
`npm ci` is installing dependencies; why does it need an RPC auth token? Can we remove this `env:` block?

Reply:
> `npm ci` is installing dependencies; why does it need an RPC auth token? Can we remove this `env:` block?

@NikolasHaimerl PTAL ☝🏻